dvc/repo/get_url.py

```diff
@@ -14,6 +14,6 @@ def get_url(url, out=None):
 
     out = os.path.abspath(out)
 
-    dep, = dependency.loads_from(None, [url])
-    out, = output.loads_from(None, [out], use_cache=False)
+    (dep,) = dependency.loads_from(None, [url])
+    (out,) = output.loads_from(None, [out], use_cache=False)
```

Comment:
Again, these are unrelated changes. Let's keep them this time, but please don't mix stuff next time. Better submit a separate PR.